.gitmodules

```diff
@@ -28,8 +28,8 @@
 	branch = master
 [submodule "platforms/nuttx/NuttX/nuttx"]
 	path = platforms/nuttx/NuttX/nuttx
-	url = https://github.com/PX4/NuttX.git
-	branch = px4_firmware_nuttx-10.1.0+
+	url = https://github.com/tiiuae/nuttx.git
+	branch = protected_build
```

Comment:
This will be removed as soon as https://github.com/PX4/PX4-Autopilot/pull/18212 is merged

Reply:
#18212 is merged